pandas/sparse/array.py

```diff
@@ -444,14 +451,18 @@ def _valid_sp_values(self):
         mask = np.isfinite(sp_vals)
         return sp_vals[mask]
 
-    def sum(self, axis=None, dtype=None, out=None):
+    def sum(self, axis=None, *args, **kwargs):
         """
```

Comment:
Well since `axis` is not even used in the implementation, the default can be changed then.